types/index.ts

```diff
@@ -1,3 +1,3 @@
 
-
-export {}
\ No newline at end of file
+export type { default as TypeChildren } from 'types/general/TypeChildren'
```

Comment:
should be replaced to absolute import that starts with `@/`

```suggestion
export type { default as TypeChildren } from '@/types/general/TypeChildren'
```